srcpkgs/ell/patches/musl-use-stdint-types.patch

```diff
@@ -0,0 +1,44 @@
+--- ell/dhcp6-private.h.orig	2020-11-30 08:48:22.817008408 -0500
```

Comment:
Same for this one, would be nice to upstream it (or report the issue and get suggestions on how to solve).

Reply:
Assuming they aren't including any `linux/` headers, they only have these types available because glibc is leaky, and really shouldn't count on them...

Reply:
@pbui use

```
diff --git a/ell/dhcp-server.c b/ell/dhcp-server.c
index c983ed4..144e83e 100644
--- ell/dhcp-server.c
+++ ell/dhcp-server.c
@@ -24,6 +24,7 @@
 #include <config.h>
 #endif
 
+#include <linux/types.h>
 #include <net/ethernet.h>
 #include <sys/socket.h>
 #include <arpa/inet.h>
```

instead

Reply:
Ah, I had thought of that... I've updated the patch to do this instead of modifying the headers.